runtime/src/main/java/com/tns/Runtime.java

```diff
@@ -31,7 +31,7 @@
 import com.tns.bindings.ProxyGenerator;
 
 public class Runtime {
-    private native void initNativeScript(int runtimeId, String filesPath, String nativeLibDir, boolean verboseLoggingEnabled, String packageName, Object[] v8Options, String callingDir);
+    private native void initNativeScript(int runtimeId, String filesPath, String nativeLibDir, boolean verboseLoggingEnabled, boolean isDebuggable, String packageName, Object[] v8Options, String callingDir);
```

Comment:
Passing raw objects as parameters through JNI is much less cumbersome to deal with on the native (C++) side as it won't require precaching fieldIds for each (8 in total) of the properties.